cluster-provision/k8s/1.25-ipv6/manifests/whereabouts/whereabouts-v0.6.yaml

```diff
@@ -193,48 +193,6 @@ subjects:
   name: whereabouts
   namespace: kube-system
 ---
-apiVersion: batch/v1
```

Comment:
1.25-ipv6 doesn't exists, so please rebase according needs
Interesting that a file rename doesn't warn about conflict once the original file was deleted

Reply:
Good point, thanks, rebased.